engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -140,43 +139,25 @@ class SpelExpression(parsed: ParsedSpelExpression,
   }
 }
 
-class SpelExpressionParser(expressionFunctions: Map[String, Method],
+class SpelExpressionParser(// for SpelExpression creation
```

Comment:
maybe we can extract class here, so we don't have to write these comments? :)

Reply:
What do you think about replacing it with one `createEvaluationContext: () => EvaluationContext`?